frontends/android/BitBoxApp/app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    buildToolsVersion "29.0.0"
-    compileSdkVersion 29
+    buildToolsVersion "33.0.0"
+    compileSdkVersion 33
```

Comment:
How come these had to be changed this time but not the previous times?

https://github.com/digitalbitbox/bitbox-wallet-app/commit/3d361b4891e90edd63bbf76a85fbef1905b42905
https://github.com/digitalbitbox/bitbox-wallet-app/commit/bba9423ae362f25ba60e351573bef582b8ad5db8

Reply:
Android Studio was warning about having `compileSdkVersion` lower than `targetSdkVersion`. `buildToolsVersion` follows `compileSdkVersion`.